pkg/core/namespace_status.go

```diff
@@ -65,3 +78,29 @@ type NamespaceStatusMultiparty struct {
 	Enabled   bool                 `ffstruct:"NamespaceStatusMultiparty" json:"enabled"`
 	Contracts *MultipartyContracts `ffstruct:"NamespaceStatusMultiparty" json:"contract,omitempty"`
 }
+
+type NamespaceMultipartyStatusOrg struct {
+	Status                NamespaceRegistrationStatus `ffstruct:"NamespaceStatusOrg" json:"status"`
+	RegistrationMessageID *fftypes.UUID               `ffstruct:"NamespaceStatusOrg" json:"registrationMessageId,omitempty"`
+}
+
+type NamespaceMultipartyStatusNode struct {
+	Status                NamespaceRegistrationStatus `ffstruct:"NamespaceStatusNode" json:"status"`
+	RegistrationMessageID *fftypes.UUID               `ffstruct:"NamespaceStatusNode" json:"registrationMessageId,omitempty"`
+}
+
+type NamespaceMultipartyStatus struct {
+	Enabled   bool                                 `ffstruct:"NamespaceMultipartyStatus" json:"enabled"`
+	Org       NamespaceMultipartyStatusOrg         `ffstruct:"NamespaceMultipartyStatus" json:"org"`
+	Node      NamespaceMultipartyStatusNode        `ffstruct:"NamespaceMultipartyStatus" json:"node"`
+	Contracts *MultipartyContractsWithActiveStatus `ffstruct:"NamespaceMultipartyStatus" json:"contracts,omitempty"`
+}
+
+type MinimalListenerCheckpoint struct {
+	Block int64 `json:"block"`
```

Comment:
We've somewhat avoided the term "block" outside of specific plugins like the Ethereum plugin (have generally used more general terms like "event" and "offset" and "checkpoint"). I don't if anything else is more appropriate here, but just pointing it out for consideration.

Reply:
The term exists inside a couple of the blockchain connectors but there is no need to let it bleed out here - I'll tidy up.